bokehjs/src/lib/models/plots/state_manager.ts

```diff
@@ -50,20 +52,24 @@ export class StateManager {
     this.changed.emit()
   }
 
-  undo(): void {
+  undo(): StateInfo|null {
```

Comment:
Space is needed around operators (here and below).

Reply:
We need a linter that complains about that, then.